website/docs/advanced-concepts/access-control/roles.md

```diff
@@ -0,0 +1,247 @@
+| **Export**         |Grants users permission to export applications in a workspace. | [App Resources](#application-resources-permissions) | 
+| **Invite**         |Grants users permission to invite users to a user group | [Groups and roles](#groups-and-roles-permissions)     |
+| **Make public**    |Grants users permission to make the applications public in a workspace |[App Resources](#application-resources-permissions)|
+| **Execute**        |Grants users permission to execute queries on a datasource | [Datasource and queries](#datasource-and-queries-permissions) |
+| **Remove users**   |Grants users permission to remove a user from a group| [Groups and roles](#groups-and-roles-permissions) |
+
+Let's understand these permissions in detail.
+
+### Application resources permissions
+
+On an application level, here are the permissions that can be assigned to user or a group -
+
+#### Create
+
+With this permission, users can create applications, pages, API queries and JS Objects. This also automatically enables permission to view, edit, and delete existing pages, database/API queries and JS Objects. Under Application Resources, you can control the **Create** permission at different hierarchical levels:
+
+##### Workspace 
+
+When you grant the **Create** permission to a workspace, it provides access to create resources only in that particular workspace, for example, create applications, pages, API queries and JS Objects. To enable permission to create workspaces, go to the [Others](#others-permissions) tab.
+
+
+
+##### Application
+
+You can restrict access to specific applications and their resources in a workspace by checking the box next to the application name.
+
+##### Page
+
+At a more granular level, you can restrict **Create** permission to a specific page and its resources within an application by checking the box next to the page name.
+
+#### Edit
+
+**Edit** allows the user to modify the existing applications, pages, database/API queries and JS Objects in a workspace. You can edit the app name, pages, and their order, color, icon, theme etc. Edit also enables access to deploy an app and connect to a Git repository. 
+Edit permission can be given at different hierarchical levels -
+
+##### Workspace 
+
+When you grant the **Create** permission to a workspace, it provides access to edit resources only in that particular workspace. This also automatically enables permission to view, edit the existing pages, database/API queries and JS Objects.
+
+
+##### Application
+
+If you want the user to have edit access only to a specific application in a workspace, open the respective workspace dropdown and check the edit permission adjacent to the respective application. **Edit** for just an application allows you to edit resources like page, actions (APIs, queries, JSObjects), Git connection etc. 
```

Comment:
workspace `tree`

Reply:
**Edit** for just an application allows you to edit resources like page, actions (APIs, queries, JSObjects), Git connection etc. `inside that application`